lang/zh-CN.json

```diff
@@ -304,5 +304,9 @@
   "account_exists": "Account exists with different credential - Login to link both accounts",
   "confirm": "确认",
   "new_version_found": "检测到新版本，刷新就会自动更新哟。",
-  "size": "大小"
-}
+  "size": "大小",
+  "team_text_owner": "拥有者",
+  "team_text_editor": "编辑",
+  "team_text_viewer": "仅查看",
+  "select_team": "选择团队"
+}
```

Comment:
It seems that the pre-commit hook will remove the last line 🤔

Reply:
That's not an issue.